src/v2/repositories/implementations/ZkBridgeRepository.ts

```diff
@@ -39,34 +39,34 @@ export class ZkBridgeRepository implements IZkBridgeRepository {
       data,
     };
   }
+
   public async getBridgeAssetData({
     param,
     web3,
   }: {
     param: ParamBridgeAsset;
     web3: Web3;
   }): Promise<TransactionConfig> {
-    const contractAddress = EthBridgeContract[param.fromChainName];
-    // ABI: https://github.com/0xPolygonHermez/zkevm-bridge-ui/blob/7c84791d06770569d316f27d62c3989bef81be58/abis/bridge.json
-    const contract = new web3.eth.Contract(ZK_EVM_BRIDGE_ABI as AbiItem[], contractAddress);
+    if (param.destNetworkId === undefined || param.destNetworkId === null) {
+      throw Error('destNetworkId is not set');
+    }
 
-    const isToL1 =
-      param.toChainName === EthBridgeNetworkName.Ethereum ||
-      param.toChainName === EthBridgeNetworkName.Sepolia;
+    const contractAddress = EthBridgeContract[param.fromChainName];
+    const isAggregateContract =
+      contractAddress === EthBridgeContract[EthBridgeNetworkName.Ethereum];
 
+    const abi = isAggregateContract ? ZK_EVM_AGGREGATED_BRIDGE_ABI : ZK_EVM_BRIDGE_ABI;
+    // ABI: https://github.com/0xPolygonHermez/zkevm-bridge-ui/blob/7c84791d06770569d316f27d62c3989bef81be58/abis/bridge.json
```

Comment:
Should we keep these comments in?  Do they serve a purpose?

Reply:
Because I can't find the ABI on Etherscan. Perhaps I should update the comment to point at the main branch.
https://etherscan.io/address/0x2a3dd3eb832af982ec71669e178424b10dca2ede#code